mw-webapp/src/component/tooltip/PositionTooltip.ts

```diff
@@ -0,0 +1,18 @@
+export const enum PositionTooltip {
+  /**
+  * Position Tooltip is top relative cursor
+  */
+  TOP = "tooltipTop",
+  /**
+  * Position Tooltip is left relative cursor
+  */
+  LEFT = "tooltipLeft",
+  /**
+  * Position Tooltip is right relative cursor
+  */
+  RIGHT = "tooltipRight",
+  /**
+  * Position Tooltip is bottom relative cursor
```

Comment:
1. does your tooltip appears relatively to cursor? Or relatively to element under cursor?
2. PositionTooltip is a public member too
3. Why not to use same key and value, like TOP = 'top' or even TOP = 'TOP'

Reply:
1. Done
2. I don't understand, what do you mean?
3. Done

Reply:
2. All public members should be documented. You document fields in this interface but not interface itself.